drivers/at86rf215/at86rf215_getset.c

```diff
@@ -204,6 +204,52 @@ int8_t at86rf215_get_ed_level(at86rf215_t *dev)
     return at86rf215_reg_read(dev, dev->RF->RG_EDV);
 }
 
+void at86rf215_enable_rpc(at86rf215_t *dev)
+{
+    if (!(dev->flags & AT86RF215_OPT_RPC)) {
+        return;
+    }
+
+    /* no Reduced Power mode available for OFDM */
+
+#ifdef MODULE_NETDEV_IEEE802154_MR_FSK
+    if (dev->fsk_pl) {
+        /* MR-FSK */
+        at86rf215_reg_or(dev, dev->BBC->RG_FSKRPC, FSKRPC_EN_MASK);
+        return;
+    }
+#endif
+#ifdef MODULE_NETDEV_IEEE802154_MR_OQPSK
+    {
+        /* MR-O-QPSK */
+        at86rf215_reg_or(dev, dev->BBC->RG_OQPSKC2, OQPSKC2_RPC_MASK);
+    }
+#endif
+}
+
+void at86rf215_disable_rpc(at86rf215_t *dev)
+{
+    if (!(dev->flags & AT86RF215_OPT_RPC)) {
+        return;
+    }
+
+    /* no Reduced Power mode available for OFDM */
+
+#ifdef MODULE_NETDEV_IEEE802154_MR_FSK
+    if (dev->fsk_pl) {
+        /* MR-FSK */
+        at86rf215_reg_and(dev, dev->BBC->RG_FSKRPC, ~FSKRPC_EN_MASK);
+        return;
+    }
```

Comment:
`The minimal ON time t on equates to the minimal preamble length of the receiver which depends on the FSK symbol rate`, this does not seem to be taking that into account, seems like `BBCn_FSKRPCONT` and `BBCn_FSKRPCOFFT` should be configured appropriately, same for `OQPSK`.

Reply:
> The minimal ON time t on equates to the minimal preamble length of the receiver which depends on the FSK symbol rate, this does not seem to be taking that into account, seems like BBCn_FSKRPCONT and BBCn_FSKRPCOFFT 

indeed

> same for OQPSK.

Fortunately not :laughing: 

> For MR-O-QPSK, the AT86RF215 supports receive operation with reduced power consumption (RPC) during RX listen.
This can be configured with sub-register OQPSKC2.RPC set to 1.
The principle of power saving is similar to the one of MR-FSK, with the exception that the OFF and ON time cannot be configured.

Reply:
Ah I already added this as part of the [MR-FSK](https://github.com/RIOT-OS/RIOT/blob/6e68d5b3a2d6390daceae4054373746f5d375bb6/drivers/at86rf215/at86rf215_fsk.c#L387) PR.

Reply:
Then can we remove that part from the PR? and leave it in the PR introducing the feature?

Reply:
And it's gone.

Reply:
Ahh I think I wasn't very cleay hahaha, I meant removing the RPC-MR-FSK, related code from this PR(#14960) and moving it to the PR introducing MR-FSK (so #14959). I though this one would be quicker to get in.. sorry for the confusion

Reply:
Ok, I removed it here.